Eclipse QOL pass

Vasily2013 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

This is a Quality of Life (QOL) pass on the Eclipse, primarily to ensure that things like intercoms, fire extinguishers, etc, are readily available in most sections of the map.

Edit: This also cleans out some active turfs and double pipes.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

Easy access to intercomms and fire extinguishers for normal crew traversing hallways is typically a good thing.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
tweak: wall-mounted extinguishers and intercomms on eclipse should now be more easy to access quickly.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Bokkiewokkie reviewed (changes requested):
Looks good so far, the only thigns I'm seeing right now are:
1. The telecomms room is missing a fire alarm inside of it.
2. There is a fire alarm on the same tile as a light in the engineering hallway.
3. Fire alarm on a light in the med hallway.
4. Intercom on a camera in the med hallway.
5. The RnD server room has no fire alarm.
6. The hallway section south of the grav gen and north of botany need additional fire alarms.
7. I don't know if you already fixed this but last I checked the capt's quarters didn't have firedoors.

Vasily2013 commented:
- Tcomms historically doesn't have fire alarms on any maps. Same goes for the server room.
2. moved fire alarm across the hall.
3. moved fire alarm one to the right.
4. intercom moved
5. addressed in the first bullet
6. added one
7. fixed before I touched it.
